layouts/shortcodes/figure.html

```diff
@@ -41,9 +48,15 @@
 {{- $m.SetInMap "image" "align" "center" -}}
```

Comment:
Why use scratch here, and why override alignment?

Reply:
Removed.